scripts/before_plugin_rm.js

```diff
@@ -5,7 +5,13 @@ var utilities = require("./lib/utilities");
 
 module.exports = function(context) {
 
-    var platforms = context.opts.cordova.platforms;
+    const platforms = context.opts.cordova.platforms,
```

Comment:
I think this can be removed as well once we change to use `before_plugin_uninstall`.  Please verify this is in fact true by removing a different plugin and ensuring the uninstall hook for this plugin does not run.